jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -1815,86 +1813,81 @@ public static void main(final String... args) throws Exception
     {
         try
         {
-            AccessController.doPrivileged((PrivilegedExceptionAction<Void>)() ->
-            {
-                Properties properties = new Properties();
-                properties.putAll(System.getProperties());
+            Properties properties = new Properties();
```

Comment:
Because the usage of privileged actions was done in very few places, leaving without many other places that would have required it.
Given that the few ones we had were definitely not enough, I think it's better to have none.